Pmods/PmodCDC1/AD7156.c

```diff
@@ -174,11 +174,20 @@ void AD7156_BurstWriteReg(int reg, char size, int txData)
 {
 	unsigned char txBuffer[4] = {0x00, 0x00, 0x00, 0x00};
 
-	txBuffer[0] = (txData & (0xFF << 8 * (size - 1))) >> (8 * (size - 1));
-	txBuffer[1] = (txData & (0xFF << 8 * (size - 2))) >> (8 * (size - 2));
-	txBuffer[2] = (txData & (0xFF << 8 * (size - 3))) >> (8 * (size - 3));
-	txBuffer[3] =  txData & (0xFF << 8 * (size - 4));
```

Comment:
How about
for (i = 0; i < size; i++)
  txBuffer[i] = (txData >> (8 * (size - i - 1))) & 0xff;